tests/integration/tier0/inhibit-if-kmods-is-not-supported/test_kmods_not_supported.py

```diff
@@ -45,14 +48,49 @@ def test_inhibit_if_custom_module_loaded(insert_custom_kmod, convert2rhel):
     assert c2r.exitstatus != 0
 
 
-def test_do_not_inhibit_if_module_is_not_loaded(shell, convert2rhel):
+@pytest.mark.unsupported_kmod_with_envar
+def test_envar_overrides_unsupported_module_loaded(insert_custom_kmod, convert2rhel, prompt_amount=PROMPT_AMOUNT):
+    """
+    This test verifies that setting the environment variable "CONVERT2RHEL_ALLOW_UNCHECKED_KMODS"
+    will override the inhibition when there is RHEL unsupported kernel module detected.
+
+    TODO(danmyway) after merging PR619 add the reference links to test metadata.
+    Verifies issue RHELC-244
+    Verifies https://github.com/oamg/convert2rhel/pull/613/
+    """
+    insert_custom_kmod()
+    os.environ["CONVERT2RHEL_ALLOW_UNCHECKED_KMODS"] = "1"
```

Comment:
```suggestion
       os.environ["CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS"] = "1"
```
.. and on couple more places in this file.

Reply:
Fixed, thanks.